packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -39,6 +41,11 @@ export const pollAppLogs = async ({
     },
   })
 
+  if (response.status == 401) {
+    await resubscribe_callback()
```

Comment:
If we're continually getting tokens from partners then hitting 401s on the poll endpoint, this would eventually overflow the stack. That could happen if Partners was producing tokens that were already expired for instance, but shouldn't happen under any normal circumstances.

I think it's fine to just accept that risk here rather than complicating this process.